client/blocks/importer/wordpress/import-everything/pre-migration/credentials.tsx

```diff
@@ -41,15 +41,12 @@ export function Credentials( props: Props ) {
 				<SubTitle className="onboarding-subtitle--full-width">
 					{
 						// translators: %(sourceSite)s and %(targetSite)s are the site slugs - e.g. my-website.wordpress.com
-						translate(
-							'Provide your SSH server credentials to migrate %(sourceSite)s to %(targetSite)s',
-							{
-								args: {
-									sourceSite: sourceSite?.slug,
-									targetSite: targetSite?.slug,
-								},
-							}
-						)
+						translate( 'Your site %(sourceSite)s will be migrated to %(targetSite)s', {
```

Comment:
What if the sourceSite or targetSite is missing, would it break the text? If so I guess we could check it before and have a fallback text?

Reply:
Following the logic from above, the `targetSite` prop is required, and when the `sourceSite` is empty for any reason, it will not render anything since we return null.

So, we are safe here.